subprojects/gst-plugins-bad/cemtrex/gstvicondewarp.cpp

```diff
@@ -319,9 +327,10 @@ bool DewarpPlugin::calibrateLens(std::string format, int width, int height, GstC
 		m_acsInfo = info;
 		m_camera->SetACS(info);
 		m_camera->SetFiltering(IMV_Defs::E_FILTER_BILINEAR_ONSTOP);
+		GST_ERROR("Error in calibration. [result: %lu]", result);
```

Comment:
I think this line is wrong, it prints error trace in a non-error case

Reply:
Addressed.